### DISCUSSION TITLE
Political Agendas feedback based on text only

### DISCUSSION THREAD

**Discussion by kevinb9n** [bafolts/terraforming-mars#2123] [General]
I have rules questions, which I think should probably lead to tweaks to the textual descriptions wherever they exist. (Some seem to just complain about wording, but I'm also checking that they do work they way I expect.)

M P2: (MarsFirst Policy 2) "After you play a… card" is a wording used on Media Group, but even it does not work that way; it should have been "When" and I hope this works the same way!
M P2 again: similarly, all "when you play a &lt;tagname> card" in canon are defined as actually meaning "when you play a &lt;tagname> tag" and all wording and behavior ought to be standardized that way, before tomorrow some community project card has two building tags.
S P3: This does mean draw 1 card per step raised, right?
U P4: Should keep seem wording as Shuttles etc. (except keep "tag" not its mistake of "card"; both that and this should be giving you -4 if there were a two-space-tag card)
R P2: is this trying to be a genuine "after" effect, as in you must carry out all the consequences of placing the tile first? I see that that would make it a more strongly negative effect, but I think it's unprecedented in canon. (or is it?)
R P3: I think that currently anything that refers generically to a "global parameter" is always including oceans; but here I think it needs to be called out one way or the other.
R P4: (example of that. I assume this includes oceans, but here I assume it strongly enough it doesn't necessarily need to be called out.)
K P2: per step raised? (I hope so.)

While I'm here I will drop my usual wording nitpicks. They aren't really important but they might just make it look more standard, more like an official expansion. Like, following the FryxGames "style guide".

General: Only one card in canon says "whenever", they're all "when"
General: TR uses raise/increase terminology instead of gain
S P4: Reason this isn't "Your Science tag requirements are -1 tag"? (consistency)
U B1 (and G B1): "or" instead of "and" make it even slightly more clear that it doesn't mean per set of the three
G P4: this would be "to any card", or to be more clear "to any one card"?

On the page listing all these things, to me having Action:/Effect: before each one would have helped reading comprehension. Or, while that's my preference, otherwise at least having it before the actions?

And it marks which ones are "repeatable", but repeatable is the assumption for communal actions.

Stopping... :-)

**Comment by nwai90** [comment]
`M P2`: You gain a rebate of 1 MC after playing a card that has a Building tag. The wording is similar to Media Group. It is not a 1 MC discount where you can pay less upfront. For now it is not a major issue as no official project card has two building tags yet. Mining Guild which is a corporation will not be affected by this Turmoil bonus.

`S P3 / K P2`: Yes, it is per step raised. With official cards like Topsoil Contract, Meat Industry etc., it is well understood even though the former does not say "gain 1 MC per microbe added" for instance. I will add it to the documentation

`U P4`: This could be updated, probably should be done separately with Shuttles / Space Station etc. if at all, rather than in isolation. For now it is not a major issue as no official project card has two Space tags yet

`R P2`: It is an after effect. They are the Reds party after all! The idea is similar to the official Reds policy: you cannot avoid paying those Reds taxes

`R P3 / P4`: Yes, it includes oceans and Venus (implied through iconography). Putting four icons there is overkill and too much of a squeeze imo

`General: Only one card in canon says "whenever", they're all "when"`: Updated

`General: TR uses raise/increase terminology instead of gain`: This only occurs in `R B2` where "gain" is more concise

`S P4: Reason this isn't "Your Science tag requirements are -1 tag"? (consistency)`: The current description is clearer imo

`U B1 (and G B1): "or" instead of "and" make it even slightly more clear that it doesn't mean per set of the three`: Official terminology and current app uses "and", hence I did not change it

`G P4: this would be "to any card", or to be more clear "to any one card"?`: Updated

`On the page listing all these things, to me having Action:/Effect: before each one would have helped reading comprehension. Or, while that's my preference, otherwise at least having it before the actions?`: Updated

**Comment by kevinb9n** [comment]
Awesome, thanks!


> `M P2`: You gain a rebate of 1 MC after playing a card that has a Building tag. The wording is similar to Media Group. It is not a 1 MC discount where you can pay less upfront. 

I just mean that Media Group's wording is anomalous as it is; this is a "when" effect, meaning that it's as if the card you're playing just had an extra "gain 1 MC" as one of its on-play instructions. To my knowledge, there are no "after" effects (you have to complete all the instructions and *then* it happens), except the one you're introducing below.

>For now it is not a major issue as no official project card has two building tags yet. Mining Guild which is a corporation will not be affected by this Turmoil bonus.

(This is again about `M P2`, and my claim that it should work per-tag and be written as "when...tag".) Yes, whether there is a bug here or not, of course I'm aware that there doesn't exist a trigger for the bug yet. 
 
> `S P3 / K P2`: Yes, it is per step raised. With official cards like Topsoil Contract, Meat Industry etc., it is well understood even though the former does not say "gain 1 MC per microbe added" for instance. I will add it to the documentation

True, it's just that when looking at a fan expansion you never know which invariants from canon they are deciding to break. Glad this is right.

> `R P2`: It is an after effect. They are the Reds party after all! The idea is similar to the official Reds policy: you cannot avoid paying those Reds taxes

Since no other "after" effects exist (not counting "after you pay the cost of", of course), I'm not even 100% sure what it means. Like, everything that triggers off of that tile placement has to happen first?

Everything else: cool, thanks!

**Comment by nwai90** [comment]
Ah, I see what you mean. The terms are analogous; will reword "after" to "when" 👍

**Comment by kberg** [comment]
This is done.
